src/components/atoms/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+import "./LinkButton.scss";
+
+interface LinkButtonProps {
+  href: string;
+}
```

Comment:
Do you want to extend from the default HTML tag? An example can be found in `InputField`/`Checkbox`

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/ccd0d1ee39f47f9b38680343ec34f721df49e4e5